packages/markdown-pdf/src/PdfTransformer.js

```diff
@@ -18,56 +18,56 @@ const PDFParser = require('pdf2json');
 const CiceroMarkTransformer = require('@accordproject/markdown-cicero').CiceroMarkTransformer;
 const PdfPrinter = require('pdfmake');
 const ToPdfMakeVisitor = require('./ToPdfMakeVisitor');
+const axios = require('axios');
 const fonts = {
     Courier: {
         normal: 'Courier',
         bold: 'Courier-Bold',
         italics: 'Courier-Oblique',
-        bolditalics: 'Courier-BoldOblique'
+        bolditalics: 'Courier-BoldOblique',
```

Comment:
These sort of automated (?) re-formatting of the code make it much harder to review PRs, as the substantive changes are now buried.

Reply:
Yes - looks like my code prettier made some format changes.  If this is an issue then I can try to revert this.

Reply:
I'll remember to turn off my code prettifier in future. Sorry.